src/components/BusinessView.tsx

```diff
@@ -1,13 +1,13 @@
 import DescriptionView from '@components/DescriptionView'
-import ImagesCarousel from '@components/ImagesCarousel'
 import RatingsView from '@components/RatingsView'
 import TitleView from '@components/TitleView'
 import { BusinessType, DisplayableBusiness } from '@futureproof/typings'
 import React from 'react'
 import { SafeAreaView, ScrollView, StyleSheet } from 'react-native'
+import ImagesCarousel from './ImagesCarousel'
```

Comment:
Why have you changed this from "@components/ImagesCarousel" to "./ImagesCarousel"?